src/index.html

```diff
@@ -8,6 +8,75 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <main class="main">
+      <div class="stars stars--0">
+        <span class="stars__star"></span>
+
+        <span class="stars__star"></span>
+
+        <span class="stars__star"></span>
+
+        <span class="stars__star"></span>
+
+        <span class="stars__star"></span>
```

Comment:
If siblings are single line you can put them together
```suggestion
        <span class="stars__star"></span>
        <span class="stars__star"></span>
        <span class="stars__star"></span>
        <span class="stars__star"></span>
        <span class="stars__star"></span>
```